src/plugins/scopes/ScopesApi.js

```diff
@@ -0,0 +1,474 @@
+/* @flow */
+import type Element from '../../elements/Element';
```

Comment:
`ScopesPlugin` tests are acutally testing not `plugin`, but `ScopesApi`, will rename those tests.